### PR TITLE
Add command-line options for ClientJob properties

### PR DESCRIPTION
**Pull request by mikeharder**
@halter73

### CODE REVIEW

File: src/BenchmarksDriver/Program.cs [hunk 1]
```diff
@@ -89,6 +98,24 @@ public static int Main(string[] args)
...
+                    {
+                        _clientJobs.Values.ToList().ForEach(c => c.Connections = int.Parse(connectionsOption.Value()));
+                    }
+                    if (threadsOption.HasValue())
+                    {
+                        _clientJobs.Values.ToList().ForEach(c => c.Threads = int.Parse(connectionsOption.Value()));
+                    }
+                    if (durationOption.HasValue())
+                    {
+                        _clientJobs.Values.ToList().ForEach(c => c.Threads = int.Parse(durationOption.Value()));
```
Comment (rynowak) [inline src/BenchmarksDriver/Program.cs]: `durationOption` .... `c.Threads`

Something seems fishy

Comment (mikeharder) [inline src/BenchmarksDriver/Program.cs]: Copy/paste bug.  I get what I deserve for not having unit tests.

**Comment by mikeharder** [comment]
@rynowak: :shipit:?

File: src/BenchmarksDriver/Program.cs [hunk 2]
```diff
@@ -89,6 +98,24 @@ public static int Main(string[] args)
...
                     serverJob.Sources = sources;
 
+                    // Override default ClientJob settings if options are set
+                    if (connectionsOption.HasValue())
+                    {
+                        _clientJobs.Values.ToList().ForEach(c => c.Connections = int.Parse(connectionsOption.Value()));
+                    }
+                    if (threadsOption.HasValue())
+                    {
+                        _clientJobs.Values.ToList().ForEach(c => c.Threads = int.Parse(connectionsOption.Value()));
```
Comment (rynowak) [inline src/BenchmarksDriver/Program.cs]: `c.Threads = int.Parse(connectionsOption.Value())` - another copy-paste bug

**Comment by rynowak** [comment]
:shipit: after fixing another copy-paste bug

**Pull request merged**